staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package request
+
+import (
+	"context"
+	"testing"
+	"time"
+)
+
+func TestWebhookDurationFrom(t *testing.T) {
+	tests := []struct {
+		name             string
+		admitDuration    time.Duration
+		validateDuration time.Duration
+	}{
+		{
+			name:             "check setting webhook duration",
+			admitDuration:    123,
+			validateDuration: 234,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			parent := context.TODO()
+			ctx := WithWebhookDuration(parent)
+			wd, ok := WebhookDurationFrom(ctx)
+			// for an empty audit ID we don't expect a copy of the parent context.
```

Comment:
remove this comment - seems like copy-paste error :)

Reply:
thx, done